components/dashboard/src/images.ts

```diff
@@ -4,26 +4,28 @@
  * See License-AGPL.txt in the project root for license information.
  */
 
+import bitbucket from './images/bitbucket.svg';
+import copy from './images/copy.svg';
+import exclamation from './images/exclamation.svg';
 import github from './images/github.svg';
 import gitlab from './images/gitlab.svg';
-import bitbucket from './images/bitbucket.svg';
-import terminal from './images/terminal.svg';
 import gitpod from './images/gitpod.svg';
 import gitpodIcon from './icons/gitpod.svg';
+import info from './images/info.svg';
+import terminal from './images/terminal.svg';
 import theia from './images/theia-gray.svg';
 import vscode from './images/vscode.svg';
-import copy from './images/copy.svg';
-import exclamation from './images/exclamation.svg';
 
 export {
+    bitbucket,
```

Comment:
I don't think loading all images from one module is a great idea. We put in some effort on in `App.tsx` to load only those modules needed but then as soon as an image is needed we pull in all at once.

Reply:
Agree! Will remove this file.